src/components/TooltipTrigger.js

```diff
@@ -66,7 +66,8 @@ const Tooltip = ({ side = 'bottom', type, target: targetId, children, id, delay
 
   return h(PopupPortal, [
     div({
-      id, role: 'tooltip',
+      id,
+      role: 'tooltip',
```

Comment:
Is there a reason for this line change? Or just sneaked in?

Reply:
will revert